webpack.config.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
```

Comment:
- [ ] OPTIONAL 
The `mode` option has not been set in your _webpack_ configuration. Kindly ensure to set the `mode` option for the _webpack_ configuration